draft-ietf-scitt-architecture.md

```diff
@@ -103,50 +112,59 @@ This document describes a scalable and flexible, decentralized architecture to e
+The ability of other Issuers to make Statements about an Artifact, produced from another Issuer, provides for third party validations.
+A Transparency Service may restrict access to Signed Statements through access control or Registration policies.
 However, third parties (such as Auditors) would be granted access as needed to attest to the validity of the Artifact, Subject or the entirety of the Transparency Service.
+Independent third parties may also make Statements about an Artifact, published on other Transparency Services.
 
-Trust in the Transparency Service itself is supported both by protecting their implementation (using, for instance, replication, trusted hardware, and remote attestation of a system's operational state) and by enabling independent audits of the correctness and consistency of its Registry, thereby holding the organization that operates it accountable.
-Unlike CT, where independent Auditors are responsible for enforcing the consistency of multiple independent instances of the same global Registry, each Transparency Service is required to guarantee the consistency of its own Registry (for instance, through the use of a consensus algorithm between replicas of the Registry), but assume no consistency between different Transparency Services.
+Trust in the Transparency Service itself is supported both by protecting their implementation (using replication, trusted hardware, and remote attestation of a system's operational state) and by enabling independent audits of the correctness and consistency of its Append-only Log, thereby holding the organization that operates it accountable.
+Unlike CT, where independent Auditors are responsible for enforcing the consistency of multiple independent instances of the same global Transparency Service, each Transparency Service is required to guarantee the consistency of its own Append-only Log (through the use of a consensus algorithm between replicas of the Transparency Service), but assume no consistency between different Transparency Services.
 
-Breadth of access is critical so the Transparency Service specified in this architecture cater to two types of audiences:
+Breadth of verifier access is critical.
+As a result, the Transparency Service specified in this architecture caters to two types of audiences:
 
-1. Producers: organizations, stakeholders, and users involved in creating or attesting to supply chain artifacts, releasing authentic Statements to a definable set of peers; and
-2. Consumers: organizations, stakeholders, and users involved in validating supply chain artifacts, but can only do so if the Statements are known to be authentic.
-Consumers MAY be producers, providing additional Signed Statements, attesting to conformance of various compliance requirements.
+1. **Issuers**: organizations, consumers, and users involved in creating or attesting to supply chain artifacts, releasing authentic Statements to a definable set of peers
```

Comment:
```suggestion
1. **Issuers**: organizations, stakeholders, and users involved in creating or attesting to supply chain artifacts, releasing authentic Statements to a definable set of peers
```

Reply:
I would retain stakeholders as consumers sounds contradictory to Issuers

Reply:
I just realized we added consumers to issuers, as opposed to Verifiers. I've fixed in this PR